cypress/e2e/po/pages/cluster-manager/cluster-manager-list.po.ts

```diff
@@ -18,7 +18,7 @@ export default class ClusterManagerListPagePo extends PagePo {
     super(ClusterManagerListPagePo.createPath(clusterId));
   }
 
-  static navTo() {
+  navTo() {
```

Comment:
This should stay as static and can be accessed via `ClusterManagerListPagePo.navTo`. I think this change will be the cause of some of the e2e failures